### PR TITLE
qt@5 5.15.14

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by Bo98** [comment]
Can we update `QtWebEngine` too?

**Comment by chenrui333** [comment]
doing both now.

File: Formula/q/qt@5.rb [hunk 1]
```diff
@@ -180,42 +179,6 @@ class QtAT5 < Formula
...
-  # Remove with Qt 5.15.14
-  patch do
-    url "https://invent.kde.org/qt/qt/qtbase/-/commit/1286cab2c0e8ae93749a71dcfd61936533a2ec50.diff"
-    sha256 "2fba1152067c60756162b7ad7a2570d55c9293dd4a53395197fd31ab770977d7"
-    directory "qtbase"
-  end
-
-  # CVE-2023-32763
-  # Original (malformed with CRLF): https://download.qt.io/official_releases/qt/5.15/CVE-2023-32763-qtbase-5.15.diff
-  # Remove with Qt 5.15.15
```
Comment (Bo98) [inline Formula/q/qt@5.rb:203]: This one might still apply because it says 5.15.15.

File: Formula/q/qt@5.rb [hunk 2]
```diff
@@ -255,6 +218,24 @@ class QtAT5 < Formula
     directory "qtbase"
   end
 
+  # CVE-2024-36048
+  # Original (malformed with CRLF): https://download.qt.io/official_releases/qt/5.15/CVE-2024-36048-qtnetworkauth-5.15.diff
+  # Remove with Qt 5.15.15
```
Comment (Bo98) [inline Formula/q/qt@5.rb:223]:
```suggestion
  # Remove with Qt 5.15.17
```

File: Formula/q/qt@5.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/q/qt@5.rb:203]: sounds about right.

**Pull request closed without merging**